src/sdl3/video.rs

```diff
@@ -2281,6 +2330,40 @@ impl Window {
             Err(get_error())
         }
     }
+
+    /// Sets a hit test function for the window.
+    #[doc(alias = "SDL_SetWindowHitTest")]
+    pub fn set_hit_test(
+        &mut self,
+        hit_test: impl Fn(crate::rect::Point) -> HitTestResult,
```

Comment:
```suggestion
      hit_test: impl Fn(crate::rect::Point) -> HitTestResult + 'static,
```
In case the closure captures something on the stack which would probably be lost, need to require it captures stuff on the heap.